recipe-server/normandy/settings.py

```diff
@@ -384,6 +391,7 @@ class ProductionInsecure(Production):
     CSRF_COOKIE_SECURE = values.BooleanValue(False)
     SECURE_HSTS_SECONDS = values.IntegerValue(0)
     SESSION_COOKIE_SECURE = values.BooleanValue(False)
+    DEFAULT_FILE_STORAGE = values.Value('django.core.files.storage.FileSystemStorage')
```

Comment:
This is for the docker stuff, right? I feel like we should set this by environment variable there instead of including it in this config, as it doesn't really fit with this config profile.